.github/workflows/smoke.yml

```
@@ -0,0 +1,111 @@
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18'
+          cache: 'npm'
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)